flink-annotations/src/main/java/org/apache/flink/annotation/docs/Documentation.java

```diff
@@ -59,6 +59,28 @@
 		int position() default Integer.MAX_VALUE;
 	}
 
+	/**
+	 * Annotation used on config option fields to include them in the "Table Options" section.
+	 *
+	 * <p>The {@link TableOption#execMode()} argument indicate which exec mode the config works for,
+	 * for batch, streaming or both.
+	 *
+	 */
+	@Target(ElementType.FIELD)
+	@Retention(RetentionPolicy.RUNTIME)
+	@Internal
+	public @interface TableOption {
+
```

Comment:
remove this line, following the existing code style in this class

Reply:
Fixed it.